manifests/09-prometheus-rules.yaml

```diff
@@ -11,18 +11,24 @@ spec:
     rules:
     - alert: ImageRegistryMissingStorageConfiguration
       expr: max_over_time(image_registry_operator_storage_configured[48h]) == 0
+      labels:
+        severity: warning
       annotations:
         message: |
           Image Registry misses a backend storage configuration. This alert
           will auto resolve once Image Registry is provided with a valid
           backend configuration.
     - alert: ImageRegistryDegraded
       expr: image_registry_operator_status_degraded == 1
+      labels:
```

Comment:
this should have a for clause, a single sample with this value should not cause an alert

Reply:
This is a very good point indeed. Thanks for catching this.